CodeGeneration/Sources/SyntaxSupport/Node.swift

```diff
@@ -208,6 +208,24 @@ public struct LayoutNode {
       preconditionFailure("NodeLayoutView must wrap a Node with data `.layout`")
     }
   }
+
+  public var grammar: SwiftSyntax.Trivia {
+    guard !children.isEmpty else {
+      return []
+    }
+
+    return docCommentTrivia(
+      from: """
+        ### Grammar
+
+        \(GrammarGenerator.grammar(for: children))
+
+        ### Children
+
+        \(GrammarGenerator.childrenList(for: children))
```

Comment:
The list of children should already be visible through DocC, under the type/protocol's _Instant Property_ section. Does it really need to be repeated in the main description too?

Reply:
I think it does because these are sorted in source order while the *Instance Properties* are sorted alphabetically.

Reply:
OK I think this is a valid reason. Although, maybe in the long term we should have the ability to specify to DocC the ordering of properties.

Reply:
Yes, if we could do that,  I agree that we might be able to remove the children list.